pkgs/tools/misc/thumbs/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+let
+  version = "0.7.1";
+in
+rustPlatform.buildRustPackage {
+  pname = "thumbs";
+  version = version;
```

```suggestion
rustPlatform.buildRustPackage rec {
  pname = "thumbs";
  version = "0.7.1";
```
This is more common in nixpkgs, but not really enforced if you dislike `rec`